internal/provider/provider.go

```diff
@@ -118,7 +120,17 @@ func (p *LxdProvider) Schema(_ context.Context, _ provider.SchemaRequest, resp *
 						"password": schema.StringAttribute{
 							Optional:    true,
 							Sensitive:   true,
-							Description: "The password for the remote.",
+							Description: "The trust password to authenticate to the LXD remote.",
+						},
+
+						"token": schema.StringAttribute{
+							Optional:    true,
+							Sensitive:   true,
+							Description: "The token to authenticate to the LXD remote",
```

Comment:
One last nit: please update that description with the one from doc.

Reply:
Is the description in docs accurate enough, or would you like me to further expand it?
As now there is a link to the documentation, I assumed it is enough to emphasize that password/token is used for initial authentication.

Reply:
The description in the doc is perfect, I'd probably reuse it as-is but strip the link/markdown syntax.